docker/Dockerfile

```diff
@@ -15,6 +15,9 @@ ARG COMMIT_SHA
 ARG TELEMETRY
 ARG BUILD_DATE
 
+# This is used to checkout the correct branch on the pgvector repo
+ARG PGVECTOR_BRANCH=v0.5.1
```

Comment:
instead of this, can we simply install w/

```bash
RUN apt-fast update && apt-fast install -y --no-install-recommends \
    "postgresql-${PG_VERSION_MAJOR}-pgvector" \
    && rm -rf /var/lib/apt/lists/*
```

Reply:
Yep, that would be much simpler. I just wasn't sure if you still wanted to maintain/build the dependencies locally similar to how it worked inside https://github.com/paradedb/paradedb/blob/v0.5.1/scripts/build_and_deploy_third_party_pg_extensions.sh before v0.5.2

Reply:
Yeah, I think using `apt` is ideal actually. We were reinventing the wheel before